airflow/ui/src/layouts/BaseLayout.tsx

```diff
@@ -0,0 +1,34 @@
+/*!
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { Box } from "@chakra-ui/react";
+import { Outlet } from "react-router-dom";
+
+import { Nav } from "./Nav";
+
+export const BaseLayout = () => {
+  return (
+    <div>
```

Comment:
Does this need to be a flexbox?

Reply:
Actually, it worked better as a div. Could adjust in the future if that stops being true.